src/components/creditcard/CreditCard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components'
+import './CreditCard.css'
+
+function CreditCard(props) {
```

Comment:
In matter of styling it doesn't look very pretty, although all the information is there and working so 👍

Reply:
Yeaah that's what I thought, rougly there. But focus on code instead of styling. You know